chicago/bills.py

```diff
@@ -86,7 +91,7 @@ def parseSearchResults(self, page) :
 
     def scrape(self):
 
-        for i, page in enumerate(self.searchLegislation()) :
+        for page in self.searchLegislation(created_after=datetime.datetime(2014, 1, 1), created_before=datetime.datetime(2014, 2, 1)) :
```

Comment:
I'm pretty sure the right way to do this is through the command line, but I kinda forget how to do that @paultag.